indico/modules/events/reminders/templates/emails/event_reminder.html

```diff
@@ -0,0 +1,80 @@
+{% extends 'emails/base_i18n.html' %}
+{% from 'events/reminders/emails/_agenda.html' import render_agenda %}
+
+{% set address = event.address %}
+{% set location = event.venue_name %}
+{% set room = event.room_name %}
+{% set label = ' {%s}'|format(event.label.title) if event.label else '' %}
+
+
+{% block subject -%}
+    [{% trans %}Event reminder{% endtrans %}] {{ event.title }}{{ label }} ({{ event.start_dt | format_datetime('short', timezone=event.tzinfo) }} {{ event.timezone }})
+{%- endblock %}
+{% block header %}{% endblock %}
+{% block footer_url %}{{ url }}{% endblock %}
+
+
+{% block body -%}
+<p>
```

Comment:
Since this is an HTML email, you can indent things properly here w/o having to worry about plaintext indentations, blank lines, etc.